server/services/stats/side/test.go

```diff
@@ -40,9 +42,32 @@ func NewTestConfig() Config {
 		Enabled:              true,
 		GoogleAnalyticsIds:   map[string]string{"TheQuestion": "ga:91655992"},
 		NewRelicApiKey:       "test",
+		TcUrl:                "http://localhost:8111",
+		TcUsername:           "root",
+		TcPassword:           "064695",
 	}
 }
 
+type testTcClient struct {
+}
+
+func NewTestTcClient() TcClient {
+	return &testTcClient{}
+}
+
+func (t *testTcClient) TcGetBuilds() ([]*schema.TcBuild, error) {
```

Comment:
this function is a mock for real teamcity in tests - you don't need to test your internet connection, you need this function to build tests of modules that have depencency on it (dashboard for example)